src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -274,27 +274,25 @@ const PagedTable: FC<Props> = ({result, properties}) => {
   }, [result, offset, height])
   const tables = useMemo(() => {
     return subsetResult(result, offset, size, properties.showAll)
-  }, [result, offset, size])
+  }, [result, offset, size]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Disabling lint ruling here because the warning ask for the `properties.showAll` to be added. Currently, `properties.showAll` doesn't change in the UI, but if it was, the process of handling it (like the other visualization components) would re-render the visualization.

![image](https://user-images.githubusercontent.com/39343323/128389725-3d73350d-02f3-46de-adfe-82c5fdc3ace4.png)